src/main/java/seedu/bigpp/ui/UI.java

```diff
@@ -1,18 +1,22 @@
 package seedu.bigpp.ui;
 
-import seedu.bigpp.pc.PCList;
-
+import seedu.bigpp.menu.MainMenu;
+import seedu.bigpp.menu.PCViewerMenu;
+import seedu.bigpp.menu.TutorialMenu;
+import seedu.bigpp.menu.ComponentMenu;
+import seedu.bigpp.menu.BuilderMenu;
 import java.io.PrintStream;
 import java.util.Scanner;
+
 public abstract class UI {
-    private static UIState uiState = UIState.MAIN_MENU;
+    
+    public static final PrintStream OUT = System.out;
```

Comment:
with this change, PCList will need change for OUT as well, need to merge most current code to your branch and check if its up to date and will run

Reply:
OUT is changed to public so that it can be accessed by other classes. I have merged team master to my branch and code passes the checks. Is that okay?